Core/scripts/dirac-install.py

```diff
@@ -1113,18 +1113,18 @@ def getExtenalsVersion(self, release=None):
 
   def getDiracOSExtensionAndVersion(self, diracOSVersion):
     """
-    This method return the diracos and version taking into 
+    This method return the diracos and version taking into
```

Comment:
Maybe just explain that the tar file looked for will be `<Extension<diracos-<version>`. And you may want to add that option at the very top of the file, where you give the examples ?

Reply:
right.